src/_posts/compliance/2000-01-01-hds.md

```diff
@@ -38,6 +38,12 @@ Please always refer the current page to get the latest compliance information
 
 [Download the current HDS 1.1 certificate](https://cdn.scalingo.com/documents/20220912-scalingo-hds-lne-certificate-38436-rev0.pdf)
 
+# 🇬🇧 What is a "Point of Contact health Professional" ("Point de contact professionnel de santé" in French)? {#en-hds-point-of-contact-health-professional}
+
+The HDS certification enjoins Scalingo to maintain a list of *Points of contact health professional* for each of its customer.
```

Comment:
```suggestion
The HDS certification enjoins Scalingo to have a up-to-date *Health Professional Point of Contact* for each of its customers.
```

Reply:
Fixed in https://github.com/Scalingo/documentation/pull/1943/commits/3943b211ac12226b8a28cb9e5049bf091371b1f2